Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
```

Comment:
Consider specifying the version of the Go tools you are using to ensure compatibility. This will help maintain consistency in environments, especially for CI/CD pipelines.